examples/websockets-http2/src/main.rs

```diff
@@ -75,7 +75,7 @@ async fn ws_handler(
                 res = ws.recv() => {
                     match res {
                         Some(Ok(ws::Message::Text(s))) => {
-                            let _ = sender.send(s);
+                            let _ = sender.send(s.to_string());
```

Comment:
This use case should show the shared bytes reuse, something like using `Utf8Payload::share`.

Reply:
This could probably be left as a follow up task, if so maybe just add a TODO here?

Reply:
An issue usually works better than a fixme comment for stuff like this. Feel free to open one.